tests/app_setup/test_db_setup.py

```diff
@@ -1,42 +1,58 @@
-from flask import current_app as app
-from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy import create_engine
 import tests.factories as factories
 
 from app.serotracker_sqlalchemy import db_tables
-from tests.utils import delete_records
+from tests.utils import delete_records, delete_all_records, insert_full_recordset
 from app.serotracker_sqlalchemy.models import DashboardSource, ResearchSource, AntibodyTarget, AntibodyTargetBridge, \
-    City, CityBridge, State, StateBridge, TestManufacturer, TestManufacturerBridge
+    City, CityBridge, State, StateBridge, TestManufacturer, TestManufacturerBridge, Country
 
 
-def test_attached_db(db):
-    assert isinstance(db, SQLAlchemy)
-
-
-def test_db_table_names():
-    engine = create_engine(app.config['SQLALCHEMY_DATABASE_URI'])
+def test_db_table_names(db):
     for table in db_tables:
-        assert table in engine.table_names()
+        assert table in db.engine.table_names()
 
 
 def test_empty_db(session):
-    with session() as _session:
-        assert len(_session.query(DashboardSource).all()) == 0
-        assert len(_session.query(ResearchSource).all()) == 0
-        assert len(_session.query(AntibodyTarget).all()) == 0
-        assert len(_session.query(AntibodyTargetBridge).all()) == 0
-        assert len(_session.query(City).all()) == 0
-        assert len(_session.query(CityBridge).all()) == 0
-        assert len(_session.query(State).all()) == 0
-        assert len(_session.query(StateBridge).all()) == 0
-        assert len(_session.query(TestManufacturer).all()) == 0
-        assert len(_session.query(TestManufacturerBridge).all()) == 0
-
-
+    assert session.query(DashboardSource).count() == 0
```

Comment:
Wouldn't this be better done iterating over the list of tables in the database? This is fragile because it breaks if we add or remove a table

Reply:
Agreed, can take an approach similar to what I described above